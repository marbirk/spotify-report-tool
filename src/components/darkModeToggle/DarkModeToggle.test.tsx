import React from 'react'
import { render } from '@testing-library/react'

import DarkModeToggle from './DarkModeToggle'

describe('DarkModeToggle', () => {
    test('renders correctly', () => {
        const { container, getByText } = render(<DarkModeToggle />)
        expect(getByText('â˜€')).toBeInTheDocument()
        expect(container).toMatchSnapshot()
    })
})
