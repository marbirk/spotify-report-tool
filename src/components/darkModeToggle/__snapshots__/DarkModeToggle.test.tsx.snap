// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DarkModeToggle renders correctly 1`] = `
<div>
  <div
    class="flex mt-0 ml-auto dark-mode-toggle"
  >
    <button
      class="text-lg"
      type="button"
    >
      ☀
    </button>
    <span
      class="flex relative items-center pt-0 pr-1 pl-1"
    >
      <input
        class="bg-gray-500 relative w-10 h-3 rounded-md cursor-pointer appearance-none outline-none dmcheck"
        data-testid="toggle"
        id="dmcheck"
        type="checkbox"
      />
      <label
        for="dmcheck"
      />
    </span>
    <button
      class="text-lg"
      type="button"
    >
      ☾
    </button>
  </div>
</div>
`;
